client/rest-high-level/src/test/java/org/elasticsearch/client/RequestConvertersTests.java

```diff
@@ -1913,6 +1914,16 @@ public void testGetTemplateRequest() throws Exception {
         assertThat(request.getEntity(), nullValue());
     }
 
+    public void testGetStoredScriptRequest() {
+        GetStoredScriptRequest storedScriptRequest = new GetStoredScriptRequest("x-script");
+        Map<String, String> expectedParams = new HashMap<>();
+
+        Request request = RequestConverters.getStoredScript(storedScriptRequest);
+        assertThat(request.getEndpoint(), equalTo("/_scripts/" + storedScriptRequest.id()));
+        assertThat(request.getParameters(), equalTo(expectedParams));
+        assertThat(request.getEntity(), nullValue());
+    }
+
     private static void assertToXContentBody(ToXContent expectedBody, HttpEntity actualEntity) throws IOException {
         BytesReference expectedBytes = XContentHelper.toXContent(expectedBody, REQUEST_BODY_CONTENT_TYPE, false);
         assertEquals(XContentType.JSON.mediaTypeWithoutParameters(), actualEntity.getContentType().getValue());
```

Comment:
add a test for deleteScript here as well?

Reply:
Thanks. Good catch